src/Stryker.Core/Stryker.Core/Initialisation/ProjectInfo.cs

```diff
@@ -21,6 +21,8 @@ public class ProjectInfo
         /// </summary>
         public FolderComposite ProjectContents { get; set; }
 
+        public string MutantControlNameSpace { get; set; }
```

Comment:
The MutantControlNameSpace is not part of the project the user is running stryker on so ProjectInfo does not seem like the correct place for this property.

Reply:
ProjectInfo is the only structure that is used all along the workflow. So the alternative is to create another structure to store some variables that relates to the current mutation session.
Note that this could be a project dependant variables, since its role is to avoid collision with project existing namespaces.

Reply:
Mhm I agree that it seems like the only suitable place because it is the only thing move all along the workflow, but it is distinctly not something part of the original project and rather something that we generated to be able to execute the mutation test. Will need to think about this one with @richardwerkman 

Introducing a new structure just for this one property also seems unfortunate..

Reply:
What if generating the random namespace was part of the 'project analysis' phase?

Reply:
It would change the scope of ProjectInfo to something beyond just info about the project we'll be testing, but perhaps it's the best option